build/kactlprocessor.py

```diff
@@ -52,6 +52,21 @@ def addref(caption, outstream):
     with open('header.tmp', 'a') as f:
         f.write(caption + "\n")
 
+def find_helper(source, start, subs):
+    first = (-1, 0)
+    for s in subs:
+        i = source.find(s, start)
+        if i != -1 and (i < first[0] or first[0] == -1):
+            first = (i, len(s))
+
+    return first
+
+def find_start_comment(source, start=None):
+    return find_helper(source, start, ['/**', "'''"])
+
+def find_end_comment(source, start=None):
+    return find_helper(source, start, ['*/', "'''"])
```

Comment:
is it worth supporting `"""`? should we try to pair comment end syntax with the matching start syntax?